eZ/Publish/Core/Persistence/Legacy/Tests/Content/UrlAlias/UrlAliasHandlerTest.php

```diff
@@ -2263,6 +2263,57 @@ public function testCreateCustomUrlAliasReusesHistory()
         );
     }
 
+    /**
+     * Test for the createUrlAlias() method.
+     *
+     * @covers \eZ\Publish\Core\Persistence\Legacy\Content\UrlAlias\Handler::createUrlAlias
+     * @group create
+     * @group custom
+     */
+    public function testCreateCustomUrlAliasAddLanguage()
```

Comment:
Tests in this class are obsolete, because they cover integration for SQLite only. Over time I'll remove them, but need to check if everything is covered by integration first.

Please create an integration test in `\eZ\Publish\API\Repository\Tests\URLAliasServiceTest`.

Reply:
test added